_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,114 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku"
+title:  "Presenting JS-Waku"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
+permalink: /2021-05-21-js-waku
+categories: platform
+summary: "Presenting JS-Waku"
+discuss: 
+---
+
+After almost three months working on JS-Waku,
+we thought it would be a good time to give it a proper introduction.
+
+<!-- Waku v2 is built for Status and Ethereum ecosystem -->
+
+First, let's review what is Waku v2.
+
+Waku v2 comes from a need to have a more scalable, better optimised solution for the Status app to achieve decentralised
+communications on resource restricted devices (ie, phones).
+We see the need of such solution in the broader Ethereum ecosystem, beyond Status.
+This is why are building Waku v2 as a decentralised communication platform for all to use and build on.
+If you want to read more about Waku v2 and what it aims to achieve,
+checkout [What's the Plan for Waku v2?](/waku-v2-plan).
+
```

Comment:
Briefly mention Whisper/Waku v1 Devp2p, libp2p as enabler for web resource restricted